joomla/joomla-cms
# [4.0][Atum] Refactor CSS variables and template options

Pull Request for Issue # .

### Summary of Changes

- Reduces defined CSS variables. `atum-bg-dark` variables combined to a single `primary` variable
- Remove the need for template php color helper file
- Refactors template options. Changes 'primary color' from hue format. Renames 'special' to 'secondary'
- Remove fill from logo SVG. Applies primary color to logo 
- Removed some unneeded CSS declarations (removed or changed to inherit)
- Made template color options reactive. Template colors change as you change the input....

![XPk1bcQ4FT](https://user-images.githubusercontent.com/2803503/65873359-ce68ca00-e37a-11e9-9909-cbf841570d17.gif)

### Testing Instructions
Apply this patch and run `node build.js --compile-css` and `node build.js --compile-js` to update the changed SCSS and JS. Alternatively, you can run npm i.

Navigate to admin template style settings. Check color options. General color check across template.


### Documentation Changes Required
